JobBot-server/jobbot/jobbot/src/test/java/org/dizzybot/jobbot/entities/PaymentTest.java

```diff
@@ -0,0 +1,52 @@
+package org.dizzybot.jobbot.entities;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class PaymentTest {
+    private Payment payment;
+
+    private Payment payment_0;
+
+    private User userEmployer;
+
+    private User userEmployee;
+
+    @Before
+    public void beforeEach() {
+        this.userEmployee = new User("Dizzy Bot Employee", "Password", "dizzybotemployee@email.com");
+        this.userEmployer = new User("Dizzy Bot Employer", "Password", "dizzybotemployer@email.com");
+        this.userEmployer.setAccount(new Account(15468779464L));
+        this.payment = new Payment(123.0, this.userEmployer, this.userEmployee);
+        this.payment_0 = new Payment(321, this.userEmployer, this.userEmployee);
+    }
```

Comment:
Do not directly access attribute, access attribute using getter.

Reply:
currently changed to 
    @Before
    public void beforeEach() {
        this.payee = new User(...);
        this.payer = new User(...);
        this.payer.setAccount(new Account(15468779464L));
        this.payment = new Payment(123.0, this.payer, this.payee);
        this.payment_0 = new Payment(321, this.payer, this.payee);
    } this is already the User instance that should be put in, so I don't know how to use getter here...